src/index.html

```diff
@@ -1,6 +1,15 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
The attributes in the <link> tag should start each on a new line with 2-space indentation related to the tag to keep the attributes correctly formatted.